src/Fair.xml

```diff
@@ -0,0 +1,26 @@
+<SPDX name="Fair License" identifier="Fair" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/Fair</url>
+  </urls>
+  <license>
+    <title>
+      <p>Fair License</p>
+    </title>
+    <copyright>
+      <p>&lt;Copyright Information&gt;</p>
+    </copyright>
+    <body>
+      <p>Usage of the works is permitted provided that this instrument is retained with the works, so that any
+         entity that uses the works is notified of this instrument.</p>
+      <p>DISCLAIMER: THE WORKS ARE WITHOUT WARRANTY.</p>
+    </body>
+    <optional>
```

Comment:
I do not believe this is part of the actual license, and I suspect this is just somebody's comment on it. See http://fairlicense.org. I suggest this text is entirely removed.